web/src/pages/ServerInfo/List.tsx

```diff
@@ -51,58 +53,84 @@ const ServerInfo: React.FC = () => {
         setData(list[0]);
       }
     });
+
+    fetchVersion().then(({ commit_hash, version }) => {
+      setCommitHash(commit_hash);
+      setDashboardVersion(version);
+    })
   }, []);
 
   return (
-    <PageContainer title={formatMessage({ id: 'page.serverinfo.pageContainer.title' })}>
-      <div className={styles.select}>
-        <Form form={form}>
-          <Form.Item wrapperCol={{ span: 5 }} style={{ marginBottom: 0 }} name="nodeId">
-            <Select
-              placeholder={formatMessage({ id: 'page.serverinfo.select.placeholder' })}
-              onChange={(value) => {
-                setData(
-                  nodeList.find((item) => {
-                    return item.id === value;
-                  }),
-                );
-              }}
-            >
-              {nodeList.map((item) => (
-                <Option key={item.hostname} value={item.id}>
-                  {item.hostname}
-                </Option>
-              ))}
+    <PageContainer title={formatMessage({ id: 'page.systemStatus.pageContainer.title' })}>
+      <Card title={formatMessage({ id: 'page.systemStatus.dashboardInfo' })} bodyStyle={{ padding: 0 }} style={{ marginBottom: 15 }}>
+        <div className={styles.wrap} >
+          {nodeList.length > 0 && (
```

Comment:
just to make sure that wether the dashboard `git hash` `version`  and `APISIX info` are Independent, which means: users will see the dashboard `git hash` and `version` no matter  whether `server-info` plugin is enabled or not

if so , IMHO, we should remote `nodeList.lenght > 0`

Reply:
oh yes, ping @LiteSun to modify.

Reply:
just removed, cc @liuxiran